XRTK.SpatialPersistence.ASA/Packages/com.xrtk.spatial-persistence.asa/Runtime/Providers/ASASpatialPersistenceDataProvider.cs

```diff
@@ -178,30 +178,42 @@ public async void StopSpatialPersistenceProvider()
         /// <param name="args"></param>
         private void CloudManager_AnchorLocated(object sender, AnchorLocatedEventArgs args)
         {
-            if (Guid.TryParse(args.Identifier, out var anchorGuid))
+            try
             {
-                if (!detectedAnchors.ContainsKey(anchorGuid))
```

Comment:
I was thinking on this while I slept last night, and I think the answer is no, because an anchor was searched for and found.

However, I want to update this to send the failed anchor I'd back to the client and they can decide if they want to delete it or not. Some maybe persisting the id and need to also remove it from their records and then call the delete asa function to remove it

Reply:
Implementation updated to include a new AnchorLocatedError event to return the Guid and a message for Anchors in error